### PR TITLE
fix(activity): remove address_id not compliant in activity

### PR DESCRIPTION
**Pull request by draphikas**

### CODE REVIEW

File: src/modules/activity/dto/update-activity.dto.ts [hunk 1]
```diff
@@ -1,3 +1,5 @@
+import { IntersectionType } from '@nestjs/swagger';
 import { CreateActivityDto } from './create-activity.dto';
+import { EntityId } from '../../../validations/common';
```
Comment (cchampou) [inline src/modules/activity/dto/update-activity.dto.ts:3]: Where is that file?

File: src/migrations/1698970534920-migration.ts [hunk 2]
```diff
@@ -5,7 +5,7 @@ export class Migration1698970534920 implements MigrationInterface {
 
   public async up(queryRunner: QueryRunner): Promise<void> {
     await queryRunner.query(
-      `CREATE TABLE "activity" ("id" SERIAL NOT NULL, "name" character varying NOT NULL, "description" character varying NOT NULL, "address_id" integer NOT NULL, CONSTRAINT "UQ_e0098522faf604f4f29ba54bba4" UNIQUE ("name"), CONSTRAINT "PK_24625a1d6b1b089c8ae206fe467" PRIMARY KEY ("id"))`,
+      `CREATE TABLE "activity" ("id" SERIAL NOT NULL, "name" character varying NOT NULL, "description" character varying NOT NULL, CONSTRAINT "PK_24625a1d6b1b089c8ae206fe467" PRIMARY KEY ("id"))`,
```
Comment (cchampou) [inline src/migrations/1698970534920-migration.ts:8]: Not an override migration instead?

File: src/modules/activity/dto/update-activity.dto.ts (hunk 1 above)
Comment (Rossb0b) [reply src/modules/activity/dto/update-activity.dto.ts:3]: Already was in the latest version of develop

File: src/migrations/1698970534920-migration.ts (hunk 2 above)
Comment (draphikas) [reply src/migrations/1698970534920-migration.ts:8]: This migration didnt run due to entity mismatch code, so I just rewrite it

File: src/modules/activity/dto/update-activity.dto.ts (hunk 1 above)
Comment (cchampou) [reply src/modules/activity/dto/update-activity.dto.ts:3]: Understood

**Review by cchampou — APPROVED** [review]

**Pull request merged**